packages/waku/src/lib/vite-rsc/cli.ts

```diff
@@ -75,25 +79,26 @@ export async function cli(
     await builder.buildApp();
   } else if (cmd === 'start') {
     const port = parseInt(flags.port || '8080', 10);
+    const host = flags.host || process.env.HOST || '127.0.0.1';
```

Comment:
Do we need the default? Can it be `undefined`?

Reply:
Yeah, and I guess it's not localhost. Anyway, if no `-h`, let's use the default (=`undefined`). It's not our job.